Extending/Backoffice-Tours/index.md

```diff
@@ -196,3 +196,37 @@ The image below shows the entire tree highlighted, but requires the user to clic
 ### customProperties
 
 A JSON object that is passed to the scope of a custom step view, so you can use this data in your view with `$scope.model.currentStep.customProperties`.
+
+## How to filter/disable tours being shown
+It is possible to hide/disable tours using a C# composer by adding to the TourFilters collection. Here is an example of disabling all the CMS core tours based on the alias.
```

Comment:
You say it's an example for disabling tours based on alias, right?
In the code example, it's also shown how to disable based on file and based on package - or am I completely off base here? 🤔

Reply:
Right! Just noticed your comment 😅 🙈 

So, wouldn't it make sense to mention that in the text as well? That there are 3 different ways to filter out the Tours? 🤔